src/analysis/excluded_terms_in_mondo/excluded_terms_in_mondo.py

```diff
@@ -0,0 +1,198 @@
+"""Excluded terms xrefed-in Mondo
+
+Resources
+- GitHub issue: https://github.com/monarch-initiative/mondo-ingest/issues/31
+- GitHub PR: https://github.com/monarch-initiative/mondo-ingest/pull/35
+"""
+import os
+import sys
+from argparse import ArgumentParser
+from typing import Dict, List
+
+import pandas as pd
+
+
+# Vars
+# # Static
+THIS_SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+ANALYSIS_DIR = os.path.join(THIS_SCRIPT_DIR, '..')
+PROJECT_DIR = os.path.join(ANALYSIS_DIR, '..', '..')
+TEMP_DIR = os.path.join(PROJECT_DIR, 'src', 'ontology', 'tmp')
+REPORTS_DIR = os.path.join(PROJECT_DIR, 'src', 'ontology', 'reports')
+MAPPINGS_DIR = TEMP_DIR
+SIGNATURE_FILES_DIR = REPORTS_DIR
+OUTDIR = REPORTS_DIR
+# # Config
+CONFIG = {
+    # todo: get list from mondo config yml:
+    'ontologies': ['doid', 'icd10cm', 'icd10who', 'ncit', 'omim', 'ordo'],
+    'mondo_mappings_path': os.path.join(MAPPINGS_DIR, 'mondo.sssom.tsv'),
+    'component_sig_pattern': 'component_signature-{}.tsv',
+    'mirror_sig_pattern': 'mirror_signature-{}.tsv',
+    # outpath: summary will be saved in same dir, with _summary at end of name
+    'outpath': os.path.join(OUTDIR, 'excluded_terms_xrefed_in_mondo.tsv'),
+    'save': True,  # save dataframes to disk
+}
+
+
+# Functions
+def uri_to_curie(uri: str) -> str:
+    """Takes an ontological URI and returns a CURIE. Works on the following patterns:
+    - http://.../PREFIX/CODE
+    - http://.../PREFIX_CODE"""
+    uri_list: List[str] = uri.split('/')
+    end_bit = uri_list[-1]
+    end_bit = end_bit.replace('>', '')  # sometimes URIs are wrapped in angle brackets <>
+    if '_' in end_bit:
+        curie = end_bit.replace('_', ':')
+    else:
+        curie = uri_list[-2] + ':' + end_bit
+    return curie
+
+
+def load_and_format_tsv(onto_name: str, pattern: str, _dir: str = SIGNATURE_FILES_DIR) -> pd.DataFrame:
+    """Load and format"""
+    # Get path
+    path = os.path.join(_dir, pattern.format(onto_name))
+    # Read
+    df = pd.read_csv(path, sep='\t').fillna('')
+    # Format
+    df['?term'] = df['?term'].apply(lambda x: x.strip())
+    df['?term'] = df['?term'].apply(uri_to_curie)
+    df = df.rename({'?term': 'term_id'})
+    return df
+
+
+def run(
```

Comment:
I need to add labels.

Chris recommended it'd be a good time to use OAK for this: https://incatools.github.io/ontology-access-kit/cli.html#runoak-fill-table

Reply:
@matentzn Having issues using OAK again. Understandable, as it is still in the version `0` stages. I created some issues:
- https://github.com/INCATools/ontology-access-kit/issues/211
- https://github.com/INCATools/ontology-access-kit/issues/212

Reply:
I added a TODO to use OAK in the future for this.

Reply:
Labels have been added. Resolving this, but I do want to use OAK for this in the future.